relm4/src/factory/sync/collections/vec_deque.rs

```diff
@@ -579,3 +579,21 @@ impl<C: FactoryComponent> FactoryVecDeque<C> {
         output
     }
 }
+
+impl<T: Clone + FactoryComponent> Clone for FactoryVecDeque<T>
+where
+    T::Init: From<T>,
+{
```

Comment:
Looks good overall, but I wonder whether it'd be better to use `T::Init: From<&T>` and remove the requirement `T: Clone` entirely. You might not want to clone the entire data when you just need `Init` in the end.